src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/index.html

```diff
@@ -0,0 +1,16 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% block title %}{% trans "Send out emails" %}{% endblock %}
+{% block content %}
+    <h1>{% trans "Send out emails" %}</h1>
+    <ul class="list-group large-link-group">
+        {% for view in views %}
+            <a class="list-group-item" href="{{ view.url }}">
```

Comment:
On a side note: it hurts a bit, that a `<li>` is missing, but adding it messes up bootstrap’s styling. Welp …

Reply:
oh, that was not on purpose. I'll make the outer element a div instead of a list so it's not horribly invalid